README.md

```diff
@@ -1,2 +1,25 @@
-# nanoflakes-go
-Reference implementation of nanoflakes for Go. 
+# Nanoflakes - JavaScript
+
+[![GitHub issues](https://img.shields.io/github/issues/nanoflakes/nanoflakes-go)](https://github.com/nanoflakes/nanoflakes-go/issues)
+[![License](https://img.shields.io/github/license/nanoflakes/nanoflakes-go)](https://github.com/nanoflakes/nanoflakes-go/tree/master/LICENSE)
+[![Twitter](https://img.shields.io/twitter/url?style=social&url=https%3A%2F%2Fgithub.com%2Fnanoflakes%2Fnanoflakes-go)](https://twitter.com/intent/tweet?text=Wow:&url=https%3A%2F%2Fgithub.com%2Fnanoflakes%2Fnanoflakes-go)
+
+Reference implementation of [nanoflakes](https://github.com/nanoflakes/nanoflakes) for Go.
+
+Licensed under the [MIT License](https://github.com/nanoflakes/nanoflakes-java/blob/master/LICENSE).
+
+### Installation
+
+Run the following command to install the package:
+
+```shell
+$ go get github.com/nanoflakes/nanoflakes-go
+```
+
+### Usage
+
+- Use `nanoflakes.localGenerator(epoch, generatorId)` to create a local nanoflake generator.
```

Comment:
I assume this would need to be rewritten for the Go language.

Reply:
I named the go package as "nanoflakes" so it would like `test, err := nanoflakes.LocalGenerator(1673042573079, 1000)`